server/mergin/commands.py

```diff
@@ -1,3 +1,5 @@
+from email.policy import default
```

Comment:
where is this one used?

Reply:
Seems like PyCharm auto thing...removing